### PR TITLE
Add WIP plugin to the ops repository and add hold label

### PR DESCRIPTION
**Pull request by mads-hartmann**
## Description

As part of https://github.com/gitpod-io/ops/issues/676 we discovered that Tide tried to merge draft PRs in the ops repository.

By adding the WIP plugin the do-not-merge/work-in-progress label is applied to DRAFT PRs. The Tide config for ops has been updated to exclude PRs with that label.

This follows the setup for the other repositories.

## Related Issue(s)

The issue was found as part of https://github.com/gitpod-io/ops/issues/676

## How to test

Once merged I will try to open a draft PR in the ops repo. Have someone approve it, and we should see that it no longer shows up in the Tide query [here](https://prow.gitpod-dev.com/tide).

## Release Notes

```release-note
NONE
```

## Documentation

NONE

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **mads-hartmann** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by ArthurSens — APPROVED** [review]
Thanks!

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 8b8583383d985312a8e5e8b7e57e94ebe3414e39</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitbot/pull/62#pullrequestreview-850376003" title="Approved">ArthurSens</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)~~ [ArthurSens]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
@mads-hartmann: Updated the following 2 configmaps:
 * `plugins` configmap in namespace `prow` at cluster `default` using the following files:
   - key `plugins.yaml` using file `config/plugins.yaml`
 * `config` configmap in namespace `prow` at cluster `default` using the following files:
   - key `config.yaml` using file `config/config.yaml`


<details>

In response to [this](https://github.com/gitpod-io/gitbot/pull/62):

>## Description
>
>As part of https://github.com/gitpod-io/ops/issues/676 we discovered that Tide tried to merge draft PRs in the ops repository.
>
>By adding the WIP plugin the do-not-merge/work-in-progress label is applied to DRAFT PRs. The Tide config for ops has been updated to exclude PRs with that label.
>
>This follows the setup for the other repositories.
>
>## Related Issue(s)
>
>The issue was found as part of https://github.com/gitpod-io/ops/issues/676
>
>## How to test
>
>
>Once merged I will try to open a draft PR in the ops repo. Have someone approve it, and we should see that it no longer shows up in the Tide query [here](https://prow.gitpod-dev.com/tide).
>
>## Release Notes
>
>```release-note
>NONE
>```
>
>## Documentation
>
>NONE


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
